src/components/App/AppIFrame.js

```diff
@@ -50,6 +51,7 @@ class AppIFrame extends React.Component {
   static propTypes = {
     app: AppType.isRequired,
     iframeRef: PropTypes.func,
+    identityEvents$: PropTypes.object,
```

Comment:
This makes `AppIFrame` quite specific instead of being about loading apps, it would be nice to move it outside in the future.

Reply:
We could add this to `App.js` and have it reload the iframe if it exists (?) or it is been used.

Reply:
Yes it could work 👍

If we start splitting `App.js` and `Wrapper.js` in the future, we could also have a module / component dedicated to the task of exchanging messages with the current app.